src/client/eppo-client.ts

```diff
@@ -478,6 +497,8 @@ export default class EppoClient {
       subjectKey,
       subjectAttributes,
       this.isObfuscated,
+      configFetchTime,
+      configPublishTime,
```

Comment:
Just wondering -- is it obvious that "time" implies date-time? Thoughts on renaming these?

```suggestion
      configFetchedAt,
      configPublishedAt,
```

Reply:
updated!